src/m365/aad/commands/group/group-list.ts

```diff
@@ -18,29 +18,36 @@ class AadGroupListCommand extends GraphItemsListCommand<Group>   {
   }
 
   public defaultProperties(): string[] | undefined {
-    return ['id', 'displayName', 'groupTypes'];
+    return ['id', 'displayName', 'groupType'];
   }
 
   public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
     this
       .getAllItems(`${this.resource}/v1.0/groups`, logger, true)
       .then((): void => {
-        this.items.forEach(group => {
-          if (group.groupTypes && group.groupTypes.length > 0 && group.groupTypes[0] === "Unified") {
-            group.groupTypes = ["Microsoft 365"];
-          }
-          else if (group.mailEnabled && group.securityEnabled) {
-            group.groupTypes = ["Mail enabled security"];
-          }
-          else if (group.securityEnabled) {
-            group.groupTypes = ["Security"];
-          }
-          else if (group.mailEnabled) {
-            group.groupTypes = ["Distribution"];
-          }
-        });
-
-        logger.log(this.items);
+        if (args.options.output === 'text') {
+          const allGroups: any = [];
+
+          this.items.forEach(group => {
+            if (group.groupTypes && group.groupTypes.length > 0 && group.groupTypes[0] === "Unified") {
```

Comment:
Since we defined defaultProperties, we don't need to copy all items. Just adding the `groupType` property to each item in the array would be enough.